packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -636,10 +634,14 @@ - (FLTPositionMessage *)position:(FLTTextureMessage *)input error:(FlutterError
   return result;
 }
 
-- (void)seekTo:(FLTPositionMessage *)input error:(FlutterError **)error {
+- (void)seekTo:(FLTPositionMessage *)input
+    completion:(void (^)(FlutterError *_Nullable))completion {
   FLTVideoPlayer *player = self.playersByTextureId[input.textureId];
-  [player seekTo:input.position.intValue];
-  [self.registry textureFrameAvailable:input.textureId.intValue];
+  [player seekTo:input.position.intValue
+      completionHandler:^(BOOL finished) {
+        [self.registry textureFrameAvailable:input.textureId.intValue];
```

Comment:
This isn't safe to call from arbitrary threads; it needs to be re-dispatched to the main thread.